README.md

```diff
@@ -35,6 +35,10 @@ Obtain the publishable and secret keys from the developer test mode on the Strip
 
 * Open the cloud-config.yaml file and replace the hard-coded name with your own.
```

Comment:
You could write the current $USER into the config too more dynamically. This would be a good exercise.